src/index.ts

```diff
@@ -9,10 +9,12 @@ export { copyToClipboard, createLink, createContextLink, share, uploadTemporaryI
 export { storage } from "./storage";
 export { createFeatureFlag, featureFlagIsEnabled, setFeatureFlagEnabled } from "./featureFlags";
 
+
 export * as account from "./exports/account";
 export * as device from "./exports/device";
 export * as groups from "./exports/groups";
 export * as leaderboards from "./exports/leaderboards";
+export * as chatbots from "./chatbot";
```

Comment:
Can we move this to `playpass.notifications`?

Chatbot seems like a carry over from FBInstant land, and we should design this API for both SMS and traditional device push notifications.